Packs/MandiantAdvantageAttackSurfaceManagement/Integrations/AttackSurfaceManagement/AttackSurfaceManagement.py

```diff
@@ -0,0 +1,458 @@
+
+            if not isinstance(response, dict) or not response.get('success'):
+                raise RuntimeError('Failed to retrieve issues from ASM')
+
+            yield from response.get('result', {}).get('hits')
+
+            if response.get('result', {}).get('more'):
+                params['page'] = params['page'] + 1
+            else:
+                break
+
+    def get_issue_details(self,
+                          issue_id: str) -> dict:
+        endpoint = f'issues/{issue_id}'
+
+        response = self._http_request('GET', endpoint,
+                                     headers={'PROJECT_ID': str(self.project_id)})
+
+        if not isinstance(response, dict) or not response.get('success'):
+            raise DemistoException('The ASM API was unable to return details for'
+                                   f'issue {issue_id} in project {self.project_id}')
+
+        return response.get('result') or {}
+
+    def get_notes(self,
+                  resource_type: str,
+                  resource_id: str) -> dict:
+        endpoint = f'notes/{resource_type}/{resource_id}'
+
+        response = self._http_request('GET', endpoint,
+                                     headers={'PROJECT_ID': str(self.project_id)})
+
+        if not isinstance(response, dict) or not response.get('success'):
+            raise DemistoException('The ASM API was unable to return notes for'
+                                   f'{resource_type} {resource_id} in project {self.project_id}')
+
+        return response.get('result') or []
+
+
+''' HELPER FUNCTIONS '''
+
+def helper_create_incident(issue: dict, project_id: int) -> dict:
+    severity_map = {1: 4, 2: 3, 3: 2, 4: 1, 5: 0.5}
+
+    raw_json = issue
+    raw_json['mirror_direction'] = 'Both'
+    raw_json['mirror_instance'] = demisto.integrationInstance()
+
+    issue = {
+        'name': issue.get('summary', {}).get('pretty_name'),
+        'details': json.dumps(issue, indent=4),
+        'occurred': parser.parse(issue.get('first_seen')).isoformat(),
+        'severity': severity_map.get(issue.get('summary', {}).get('severity'),),
+        'rawJSON': json.dumps(raw_json),
+        'dbotMirrorId': issue.get('id', issue.get('uid')),
+        'CustomFields': {
+            'masmcollection': issue.get('collection'),
+            'masmproject': str(project_id),
+            'masmconfirmed': issue.get('summary', {}).get('confidence') == 'confirmed',
+            'masmentityname': issue.get('entity_name'),
+            'masmcategory': issue.get('summary', {}).get('category')
+        }
+    }
+
+    return issue
+
+def fetch_incident_helper(client: Client, last_run: dict) -> typing.Tuple[dict, list]:
+    last_start_time: datetime
+
+    params = demisto.params()
+
+    if not client.project_id:
+        raise DemistoException('Must configure a Project ID to fetch incidents from Mandiant ASM')
+
+    if last_run and 'start_time' in last_run and last_run['start_time']:
```

Comment:
```suggestion
    if last_run and 'start_time' in last_run and last_run['start_time']:
```

save the last run as 'time`, it would have better compatibility in case you need [lookback](https://xsoar.pan.dev/docs/integrations/fetch-incidents-lookback)